EventKit/eventkit-cloud
# Add toasts

# Description of Improvement

Adds react-toastify to the project and sets up basic toasts around async actions. Goal is to provide more feedback to users around api success + failures that aren't caught and handled by components already.

In addition to the toasts there is a small fix around the permissions banner that was overriding some of our base text css when used due to the use of CSSBaseline component. Banner should still appear when providers with attribute class filter exist, but surrounding css on create workflow shouldn't be impacted.

## How to test

You will need to ensure new dependency is installed locally, and run collectstatic to get new files
Verify all tests pass
Verify DataProvider error toast appears on pages when backend is down (Dashboard, Create, etc)
Verify DataPack creation toast appears on successful DataPack creation

## Quality Assurance

The following items have been updated:


## Screenshots



:smile:
